### ISSUE TITLE
Which python command are you using for CIFAR and STL10 experiments?

### ISSUE THREAD

**Issue by LIUNancy** [Philip-Bachman/amdim-public#3]
Hi Philip,

Thanks for your great work. Could you please share your python command for CIFAR and STL10 experiments? The size parameters of AMDIM-large for CIFAR seem to have two different versions in Table 2 caption and body text. Which one are you using for the results reported in the paper?

Thank you very much for your help.

Kind regards,
Nancy

**Comment by wbuchwalter** [comment]
Hi Nancy,
The correct configuration should be the one in the table 2.(b), so: `--ndf 256  --n_rkhs=2048,  --n_depth=10`.
The other configuration mentioned in the body is a mistake on our side, thanks for catching it.

Closing this issue as I think it answers your question, but if not let us know.

**Comment by LIUNancy** [comment]
Thanks for your reply. And what about the other hyperparameters (such as batch size, learning rate) for STL10 experiment, are they same as the hyperparameter for ImageNet?

**Comment by wbuchwalter** [comment]
For CIFAR/STL10, you should be able to reproduce the results by using a batch size of 480 (or more if your setup permits it) and use the defaults for all the other hyperparameters (including LR).

**Comment by LIUNancy** [comment]
Do you enable amp for CIFAR/STL10 training?

**Comment by wbuchwalter** [comment]
Amp shouldn't influence the final result, but you should definitly enable it if you have a V100/P100 or other recent GPUs that support FP16.

**Comment by martinmamql** [comment]
Dear Philip and William,

Thanks for the great work. Did experiments which achieved SOTA on CIFAR 10/100 also use FastAutoAugmentation, or just standard augmentation as provided in this repository?

Thank you so much!
Martin
